Public/Sdk/Public/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -108,6 +108,23 @@ function validateArguments(args: QTestArguments): void {
     }
 }
 
+/**
+ * Find Flaky Supression file from the .config directory of source code
+ */
+function findFlakyFile(): File {
+    let configDir = d`${Context.getMount("SourceRoot").path}\.config`;
+    let flakyDir = d`${configDir}\flakytests`;
+    let flakyFileName = 'CloudBuild.FlakyTests.json';

```

Comment:
Iman, The file name is fixed. We dont allow user to change the name of the file. Also, the location, we restricted it based on the suggestion from Mike to two locations. Either in .config folder or under a subfolder .config\flakytests. Hope this helps clarify.

Reply:
I have concern with using `SourceRoot`. Office, for example, has their source root set in the build directory where the metabuild produces the specs for the product build. But users may want to have the file in their source `dev` directory. 

My preference is, as mentioned above, the file can be specified through `QTestArguments`. The caller of `runQTest` is the one who should find where the flaky file is. For example, for our selfhost, the BuildXLSDK.dsc can be the one that specifies the file should be located to the two locations Mike suggested. While for Office, they can control it with environment variable. But the QTestRunner itself is agnostic on the shape of the repo.

Or have I overlooked something?

Reply:
Discussed offline. Currently there is no direct way for QTest SDK to access files from enlistment root (in case it is different from `SourceRoot `) without running into file monitoring violations. If there is work required where we need to access enlistment root we will handle it in a later PR.